resources/js/Pages/Maintenance.vue

```diff
@@ -1,189 +1,55 @@
+                <Navbar />
             </div>
-            <p
-                class="mr-2 border border-solid border-black p-2.5 px-8 text-lg font-bold text-2023-teal shadow-md shadow-2023-teal"
-            >
-                semana_de_informática
-            </p>
-            <p class="margin-0 text-center text-2xl font-bold text-2023-teal">
-                25 a 31 de outubro
-            </p>
-            <p
-                class="m-0 text-center text-lg font-bold text-2023-teal max-ml:text-base"
-            >
-                Estamos a preparar mais uma edição da SINF!
-            </p>
-            <a
-                href="https://2022.sinf.pt"
-                class="relative m-0 text-center text-lg font-bold text-2023-teal underline max-ml:text-base"
-            >
-                Revê a edição do último ano aqui!
-            </a>
             <section
-                class="relative mx-[25%] flex content-center items-center justify-center border border-solid border-black p-8"
+                class=" grow relative flex flex-col gap-8 py-36 md:justify-evenly md:py-0"
             >
-                <p
-                    class="left-[calc(50% - 86.5px)] absolute -top-[30px] w-max border border-solid border-black bg-2023-red p-2.5 px-5 text-lg font-bold text-white decoration-white shadow-md shadow-2023-bg"
-                >
-                    Contacta-nos!
-                </p>
-                <div
-                    id="contacts-content-desktop"
-                    class="flex w-full flex-row items-center justify-evenly gap-2.5 max-ml:hidden"
-                >
-                    <a
-                        class="flex w-fit rounded-full bg-2023-red p-2"
-                        target="_blank"
-                        href="https://www.facebook.com/NIAEFEUP"
-                    >
-                        <OhVueIcon
-                            name="io-logo-facebook"
-                            fill="white"
-                        ></OhVueIcon>
-                    </a>
-                    <a
-                        class="flex w-fit rounded-full bg-2023-red-dark p-2"
-                        target="_blank"
-                        href="https://www.linkedin.com/company/nifeup/mycompany/"
+                <div class="mx-auto md:ml-36">
+                    <h1
+                        class="w-3/12 text-justify font-space-grotesk text-5xl font-bold tracking-tight text-white md:text-7xl"
                     >
-                        <OhVueIcon
-                            name="io-logo-linkedin"
-                            fill="white"
-                        ></OhVueIcon>
-                    </a>
+                        <p class="whitespace-nowrap">ENCONTRO</p>
+                        <p class="whitespace-nowrap">NACIONAL DE</p>
+                        <p class="whitespace-nowrap">ESTUDANTES DE</p>
+                        <p class="whitespace-nowrap">INFORMÁTICA</p>
+                    </h1>
```

Comment:
```suggestion
                        <p class="whitespace-nowrap uppercase">Encontro</p>
                        <p class="whitespace-nowrap uppercase">Nacional de</p>
                        <p class="whitespace-nowrap uppercase">Estudantes de</p>
                        <p class="whitespace-nowrap uppercase">Informática</p>
                    </h1>
```

Reply:
Useful for copy paste and search engine indexing

Reply:
As I understood from the image department's channel, this text is supposed to be all lowercase, is it correct?

Reply:
Yes, it means that the suggestions are wrong and that the "lowercase" class should be used instead of the "uppercase" class